### PR TITLE
JSDK-247 - Ensuring method signature compatability

### PR DESCRIPTION
**Pull request by scribe**
This is a small pull request to restore the method signature that had been overlooked in my previous pull request.

### CODE REVIEW

**Comment by rpmoore** [comment]
Out of date

File: ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/pagination/GetBucketLoaderFactory.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ * ****************************************************************************
+ */
+
+package com.spectralogic.ds3client.helpers.pagination;
+
+import com.spectralogic.ds3client.Ds3Client;
+import com.spectralogic.ds3client.models.Contents;
+import com.spectralogic.ds3client.utils.collections.LazyIterable;
+
+public class GetBucketLoaderFactory extends GetBucketKeyLoaderFactory<Contents> {
```
Comment (rpmoore) [inline ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/pagination/GetBucketLoaderFactory.java]: Add some javadoc explaining what this class is and what it does.

Comment (scribe) [inline ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/pagination/GetBucketLoaderFactory.java]: I'm concerned this isn't quite right, but it's getting closer.

File: ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/pagination/GetBucketLoaderFactory.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+package com.spectralogic.ds3client.helpers.pagination;
+
+import com.spectralogic.ds3client.Ds3Client;
+import com.spectralogic.ds3client.models.Contents;
+import com.spectralogic.ds3client.utils.collections.LazyIterable;
+
+/**
+ *  This class is a child of {@link GetBucketKeyLoaderFactory}
+ *  that provides a sane default for mapping {@link com.spectralogic.ds3client.models.ListBucketResult}
```
Comment (rpmoore) [inline ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/pagination/GetBucketLoaderFactory.java]: What are the sane defaults?

Comment (scribe) [inline ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/pagination/GetBucketLoaderFactory.java]: Added what they actually are.

**Pull request merged**